src/solace_ai_connector/components/general/llm/langchain/langchain_chat_model_base.py

```diff
@@ -109,9 +134,18 @@ def invoke(self, message, data):
 
         session_id = data.get("session_id", None)
```

Comment:
[SUG] session_id and clear_history are missing from the input schema. You can update the input schema.

Reply:
This is the base class, they are only needed for the history. They already exist on t`langchain_chat_model_with_history`